packages/schema/package.json

```diff
@@ -13,19 +13,19 @@
     "/schema.js"
   ],
   "scripts": {
-    "preversion": "git pull && yarn test && yarn build",
+    "preversion": "git pull && pnpm test && pnpm build",
     "test": "mocha -t 10s --recursive test --exit",
     "smoke-test": "mocha -t 10s --recursive smoke-test --exit",
     "test:debug": "mocha --recursive --inspect-brk test",
     "lint": "eslint lib",
     "lint:fix": "eslint --fix lib",
     "coverage": "istanbul cover _mocha -- --recursive",
     "export": "node bin/export.js && prettier --write exported-schema.json",
-    "docs": "node bin/docs.js",
-    "build": "yarn docs && yarn export",
+    "build-docs": "node bin/docs.js",
```

Comment:
Renamed `docs` to `build-docs` because `pnpm docs` seems to be reserved. When I run `pnpm docs`, it opens the doc site with a browser.